statics/js/components/flow-table.js

```diff
@@ -615,16 +616,37 @@ Vue.component('flow-table', {
     flowDetailLinks: function(flow) {
       var self = this;
 
+      var links = {};
+
+      if (flow.ANodeTID !== "*") {
+        links.ANodeTID = {
+          "class": "indicator glyphicon glyphicon-link raw-packet-link",
+          "onClick": function() {},
+          "onMouseOver": function(){ self.highlightRelatedTID(flow.ANodeTID); },
```

Comment:
this node is already highlighted (https://github.com/skydive-project/skydive/blob/master/statics/js/components/flow-table.js#L301) by this line. after mouse over and out, it is losing the highlight.
can we remove the highlight of the detailed row of flow?

Reply:
Done now the already highlighted node stay highlighted even on mouse out